ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_random.py

```diff
@@ -272,3 +273,38 @@ def test_paddle_randint_like(
         high=high,
         dtype=dtype[0],
     )
+
+
+# multinomial
+@handle_frontend_test(
+    fn_tree="paddle.tensor.random.multinomial",
+    input_dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid", full=False),
```

Comment:
No need to keep `full=False` here. Just keep `"valid"` here and rest should be dealt with automatically